pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -302,6 +302,7 @@ func (c *Controller) syncOwnedProvisionRequest(ctx context.Context, wl *kueue.Wo
 			}
 
 			if err := c.client.Create(ctx, req); err != nil {
+				c.record.Eventf(wl, corev1.EventTypeWarning, "ProvisioningRequestError", "Error creating ProvisioningRequest: %q", req.Name)
```

Comment:
```suggestion
				c.record.Eventf(wl, corev1.EventTypeWarning, "FailedCreate", "Error creating ProvisioningRequest: %q", req.Name)
```
Or `FailedToCreate`. I assume the event is attached to the ProvisiningRequest, so we don't need to repeat the type in the event reason. Inspired by https://github.com/kubernetes/kubernetes/blob/21f54c3ca1ac87611e101f46d8b05682f51efbee/pkg/controller/cronjob/cronjob_controllerv2.go#L635

Reply:
Done. Thanks